[qconv] Remove duplicate calculation of output shape

dskhudia opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#26684 [qconv] Remove duplicate calculation of output shape**

Output heights and widths are already calculated by conv_p. Remove the duplicate calculation.

Differential Revision: [D17538375](https://our.internmc.facebook.com/intern/diff/D17538375/)

dskhudia committed:
[qconv] Remove duplicate calculation of output shape

Output heights and widths are already calculated by conv_p. Remove the duplicate calculation.

Differential Revision: [D17538375](https://our.internmc.facebook.com/intern/diff/D17538375/)

[ghstack-poisoned]

supriyar commented:
@dskhudia `convOutputShape` is also used by qnnpack code below. Please don't delete it.

supriyar reviewed (changes requested):
Don't delete convOutputShape

dskhudia committed:
Update on "[qconv] Remove duplicate calculation of output shape"


Output heights and widths are already calculated by conv_p. Remove the duplicate calculation.

Differential Revision: [D17538375](https://our.internmc.facebook.com/intern/diff/D17538375/)

[ghstack-poisoned]

dskhudia commented:
@supriyar left the convOutputShape as is.
